authdelegate/entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash -e
+
+source /etc/profile.d/gvm.sh
+
+if [ "$1" = "run-server" ]; then
```

Comment:
For this entrypoint script and the others, will you ever not pass in `run-server` as a parameter?